openpype/hosts/houdini/api/lib.py

```diff
@@ -568,29 +568,58 @@ def get_template_from_value(key, value):
     return parm
 
 
-def get_frame_data(node):
-    """Get the frame data: start frame, end frame and steps.
+def get_frame_data(node, handle_start=0, handle_end=0, log=None):
+    """Get the frame data: start frame, end frame, steps,
+    start frame with start handle and end frame with end handle.
+
+    This function uses Houdini node as the source of truth
+    therefore users are allowed to publish their desired frame range.
+
+    It also calculates frame start and end with handles.
```

Comment:
> This function uses Houdini node as the source of truth
> therefore users are allowed to publish their desired frame range.

I personally wasn't sure what this meant with regards to the data.

I wonder if this would be worded clearer:

```suggestion
    """Get the frame data: start frame, end frame, steps,
    start frame with start handle and end frame with end handle.

    This function uses Houdini node's `trange`, `t1, `t2` and `t3` 
    parameters as the source of truth for the full inclusive frame 
    range to render, as such these are considered as the frame 
    range including the handles.

    The non-inclusive frame start and frame end without handles
    are computed by subtracting the handles from the inclusive
    frame range.
```